examples/hermes-agent/tester.py

```
@@ -0,0 +1,85 @@
+        print("Failed to find hermes-agent pod.")
+        sys.exit(1)
+    
+    print(f"Found pod: {pod_name}")
+    
+    # Start port forwarding using local kubeconfig if present
+    cmd = ["kubectl"]
+    if os.path.exists("kubeconfig"):
+        cmd.append("--kubeconfig=kubeconfig")
+    cmd.extend(["port-forward", f"pod/{pod_name}", "8642:8642"])
+    
+    pf_process = subprocess.Popen(cmd)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)